dockerfile_scripts/build_aws.sh

```diff
@@ -17,8 +17,8 @@ if [ "$OFI" = "1" ]; then
                 tcsh
 
   # Install AWS_OFI_NCCL
-  AWS_VER=v1.6.0
-  AWS_VER_NUM=1.6.0
+  AWS_VER=v1.4.0
```

Comment:
is this correct? going backwards to a previous version of AWS?

otherwise, looks good to me

Reply:
yeah, it seems to work better with the 1.4.0 version.